centos2almaconverter/actions/packages.py

```diff
@@ -119,7 +119,6 @@ def __init__(self, temp_directory: str):
             "libwebp7": "libwebp",
             "libzip5": "libzip",
             "libytnef": "ytnef",
-            "libidn2": "libidn2",
```

Comment:
Could you please also check if conversion pass well when the package is installed?

Reply:
Yes, I've tried the conversion with the OS vendor package installed. I've also tried it with the version from custom repo (www.city-fan.org), in which case the conversion fails due to other reasons (unavailable AL8 package versions).